.travis.yml

```diff
@@ -1,15 +1,25 @@
 language: node_js
-node_js:
```

Comment:
I think updating travis.yml must be done in another pr. Those changes are irrelevant to typing.

Reply:
the types should be tested. Would you like one big command like

```
existing_tests && tests_for_types
```
instead of travis-ci jobs that run in parallel?